.github/workflows/fork-build.yml

```diff
@@ -46,7 +46,7 @@ jobs:
       - name: Install markbind-cli@master
         if: inputs.version == 'development'
         run: |
-          npm i -g npm@8.3.1
```

Comment:
This can be removed if the default node version is 16? Or have you found out that the default npm in CI is not 8.19.3?

Reply:
Yes removed!